src/custom/pages/Faq/index.tsx

```diff
@@ -414,7 +414,10 @@ export default function Faq() {
             <LinkScrollable href={'#what-is-presign'}>pre-sign</LinkScrollable>.
           </p>
           <p>
-            Regular, non Smart Contract wallets (called EOA) require the wallet to support offchain-signing (EIP712
+            Regular, non Smart Contract wallets (called EOA) require the wallet to support offchain-signing (
+            <ExternalLink href="https://eips.ethereum.org/EIPS/eip-712" target="_blank" rel="noopener noreferrer">
```

Comment:
Question out of curiosity, what is the difference between `ExternalLink` and `a`, and why doesn't `ExternalLink` already add things like `target="_blank" rel="noopener noreferrer"`?

Reply:
It has to do with analytics and styles. We track the clicks user do on those links. `ExternalLink` adds already `target="_blank"` by default that's a good point, its redundant.

Reply:
I used this PR to simplify the links and remove the redundant tags